src/util-threshold-config.c

```diff
@@ -1082,15 +1090,19 @@ void SCThresholdConfParseFile(DetectEngineCtx *de_ctx, FILE *fp)
 
         esc_pos = SCThresholdConfLineIsMultiline(line);
         if (esc_pos == 0) {
-            rule_num++;
-            SCLogDebug("Adding threshold.config rule num %"PRIu32"( %s )", rule_num, line);
-            SCThresholdConfAddThresholdtype(line, de_ctx);
+            if (SCThresholdConfAddThresholdtype(line, de_ctx) < 0) {
+                if (RunmodeGetCurrent() == RUNMODE_CONF_TEST)
```

Comment:
I think this should include `de_ctx->failure_fatal`, which can force suri to error out when not in `-T` mode. Related config option `engine.init-failure-fatal`, commandline `--init-errors-fatal`

Reply:
Will do.